docs/reference/index.md

```diff
@@ -18,4 +18,3 @@ keywords: "Engine"
 * [Dockerfile reference](builder.md)
 * [Docker run reference](run.md)
 * [Command line reference](commandline/index.md)
-* [API Reference](api/index.md)
```

Comment:
This should be updated to https://docs.docker.com/engine/api/, which is not yet live, but will be the URL where the API documentation will be